docs/tutorials/optimizers_adamw.ipynb

```diff
@@ -0,0 +1,692 @@
+{
```

Comment:
<p>Can you remove the output of this cell. Displaying the results in graphical form below is sufficient.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/1217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>